src/terra-dev-site/doc/application/reference.1/contexts/ThemeContext.doc.mdx

```diff
@@ -0,0 +1,25 @@
+import ThemedComponent from './example/ThemedComponent?dev-site-example';
+
+# ThemeContext
+
+The ThemeContext defines an interface for access to the framework's current theme.
+
+This context can be used to apply any theme related changes to a component.
+
+The most common use would be to apply a theme class to the root tag of the component to apply theme variables. See below for an example.
+
+
+> Note: An ThemeContextProvider is rendered by [ApplicationBase](/components/terra-application/application/reference/components/application-base).
+> Any components rendered within ApplicationBase can access a ThemeContext without rendering additional providers.
+
+## Usage
+
+<ThemedComponent description="This example pulls the current theme from the theme context and adds the theme class name to apply theme variables." isExpanded />
+
+## Context Value
+
+The ThemeContext provides an object with the following values:
+
+|Key Name|Type|Is Required|DefaultValue|Description|
+|---|---|---|---|---|
+|`className`|String|optional|undefined|The current application theme className. The default theme is indicated as undefined or empty string.|
```

Comment:
> The default theme is indicated as undefined or empty string.

Can it really be either?

Reply:
by default it would be undefined if you passed nothing, but i suspect it's set to empty string in terra-dev-site

Reply:
So I guess brass tacks are that we don't control what the default theme is indicated as, but generally we expect it would be either undefined or empty-string. To have more assurance here we'd have to dictate how themes are included at the terra application layer.